design/sd-local.md

```diff
@@ -115,6 +115,7 @@ test
 
 ### Prerequisites
 - Docker runtime
+  - Have docker runtime execute permission (or you can use `--sudo` option)
```

Comment:
How about `Permission for docker runtime`?

Reply:
I fixed.